server.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 const app = express();
 const path = require("path");
-const { mongoose } = require("./config/index.js");
+/* setup config = */ require("./config/index.js");
```

Comment:
Decide whether you'll need the assignment later on! And then decide if you need the commented code :)

Reply:
Thanks! I guess I'll just remove that!